vignettes/GGIR.Rmd

```diff
@@ -734,9 +734,9 @@ Part 2 generates the following output:
 | `_M VPA_E5S_B1M80_T100`        | MVPA calculated based on 5 second epoch setting bout duration 1 Minute and inclusion criterion of more than 80 percent. This is only done for metric ENMO at the moment, and only if mvpa threshold is not left blank     |
 | `_ENMO_mg`         | ENMO or other metric was first calculated per day and then average according to AD, WD, WWE, WWD           |
 | data exclusion strategy        | A log of the decision made when calling g.impute: value=1 mean ignore specific hours; value=2 mean ignore all data before the first midnight and after the last midnight |
-| n hours ignored at start of meas (if strategy=1)        | number of hours ignored at the start of the measurement (if strategy = 1) A log of decision made in part2.R       |
-| n hours ignored at end of meas (if strategy=1)          | number of hours ignored at the end of the measurement (if strategy = 1). A log of decision made in part2.R         |
-| n hours ignored at end of meas (if strategy=1)          | number of days of measurement after which all data is ignored (if strategy = 1) A log of decision made in part2.R    |
+| n hours ignored at start of meas (if strategy=1, 3 or 5)        | number of hours ignored at the start of the measurement (if strategy = 1, 3 or 5) A log of decision made in part2.R       |
```

Comment:
`number of hours ignored at the start of the measurement` is not correct with these updates because for strategy 3 and 5 it is about the start/end of the ndayswindow.